doc/_templates/index.html

```diff
@@ -120,4 +120,8 @@ <h2>{%trans%}Contributor Guide{%endtrans%}</h2>
       <li>{%trans path=pathto("authors")%}<a href="{{ path }}">Sphinx Authors</a></li>{%endtrans%}
     </ul>
 
+    <h2>{%trans%}Code of Conduct{%endtrans%}</h2>
+
+    {%trans path=pathto("code_of_conduct")%}Please adhere to our <a href="{{ path }}">Code of Conduct</a>.{%endtrans%}
```

Comment:
Hope not to sound too nitpicky but how is one suppose to adhere to the CoC concretely?